Xamarin.Forms.Platform.MacOS/Renderers/ImageRenderer.cs

```diff
@@ -1,25 +1,30 @@
 ﻿using System;
 using System.ComponentModel;
+using System.Threading.Tasks;
 using AppKit;
-using CoreAnimation;
-using CoreGraphics;
 
 namespace Xamarin.Forms.Platform.MacOS
 {
-	public class ImageRenderer : ViewRenderer<Image, NSView>
+	public class ImageRenderer : ViewRenderer<Image, NSImageView>, IImageVisualElementRenderer
 	{
 		bool _isDisposed;
 
+		public ImageRenderer()
+		{
+			ImageElementManager.Init(this);
+		}
+
 		protected override void Dispose(bool disposing)
 		{
 			if (_isDisposed)
 				return;
 
 			if (disposing)
 			{
-				CGImage oldUIImage;
-				if (Control != null && (oldUIImage = Control.Layer.Contents) != null)
+				NSImage oldUIImage;
+				if (Control != null && (oldUIImage = Control.Image) != null)
```

Comment:
Perhaps, it will be look more clear, if change to 

```csharp
var oldNsImage = Control?.Image; // nitpick: it's NSImage, not UIImage
if(oldNsImage != null)
{
    ImageElementManager.Dispose(this);
	oldNsImage.Dispose();
}

Reply:
Ahh, it's cloned from iOS renderer, i see
https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.iOS/Renderers/ImageRenderer.cs#L41-L59